### PR TITLE
Bf/flaky tests

### PR DESCRIPTION
**Pull request by dora-korpar**
On a quest to reduce the flakiness of the external backend tests

### CODE REVIEW

**Comment by dora-korpar** [comment]
@ironman-machine try

**Comment by LaurenSpiegel** [comment]
@ironman-machine r+

File: tests/functional/aws-node-sdk/test/multipleBackend/put/put.js [hunk 1]
```diff
@@ -214,6 +221,7 @@ describe('MultipleBackend put object', function testSuite() {
             it('should encrypt body only if bucket encrypted putting ' +
             'object to AWS',
             done => {
+                const key = `somekey-${Date.now()}`;
```
Comment (nicolas2bert) [inline tests/functional/aws-node-sdk/test/multipleBackend/put/put.js]: you could also generate the key in a beforeEach like ``` this.currentTest.key=`somekey-${Date.now()}` ```
and use it in all the tests `this.test.key`

**Comment by LaurenSpiegel** [comment]
@ironman-machine r+

**Comment by LaurenSpiegel** [comment]
@ironman-machine r+

**Pull request merged**